src/qlever/qleverfile.py

```diff
@@ -414,5 +415,11 @@ def read(qleverfile_path):
         if index.get("text_index", "none") != "none":
             server["use_text_index"] = "yes"
 
+        # Add other non-trivial default values.
+        try:
+            config["server"]["host_name"] = socket.gethostname()
+        except Exception:
+            pass
+
```

Comment:
question: Why are you doing this? We hat `socket.getfqdn()` as a default value some time ago, but removed it because the call could take a long time.

Reply:
That is understandable. I am bit uncertain here. I don't know where this can break. If it breaks it is probably more confusing for users than `localhost`.

`gethostname` shouldn't take that long and I don't know of any common errors, so let's set it directly as the argument's default value (that what we did with `getfqdn`):
```python
        server_args["host_name"] = arg(
            "--host-name",
            type=str,
            default=f"{socket.gethostname()}",
            help="The name of the host on which the server listens for "
            "requests",
        )
```

Reply:
@Qup42 The problem with this construction is that if for some reason this call fails, the whole script fails. That's why I put it in a `try ... except` block in the part with the non-trivial initializations